recipes/default.rb

```diff
@@ -17,7 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if node['audit']['inspec_package_source']
+if node['audit']['inspec_package']['source']
   include_recipe 'audit::inspec_package'
```

Comment:
Curious if there is a reason to break out inspec installation into two separate recipes?  Would it be tighter to just `include_recipe 'audit::inspec'` here and let the `inspec` recipe have the conditional logic to decide whether to install from package or from gem within an if statement?